src/8a860_len_3f30.c

```diff
@@ -1,11 +1,97 @@
 #include "common.h"
 
+extern s32 D_8010D650;
+extern s32 D_8010D660;
+extern s32 D_8010D664;
+extern s8 D_8010D699;
+extern PopupMenu* D_8010D69C;
+extern s32 D_8010D640;
+extern s32 D_8010D65C;
+extern s32 D_8010D668;
+extern s32 D_8010D66C;
+extern s32 D_8010D670;
+extern s32 D_8010D674;
+extern s32 D_8010D678;
+extern s8 D_8010D69A;
+extern s32 D_8010D648;
+extern s32 D_8010D64C;
+extern s8 D_8010D654;
+extern u8 D_8010D655;
+extern u8 D_8010D656;
+extern s8 D_8010D68E;
+extern s8 D_8010D68F;
+extern s8 D_8010D691;
+extern s8 D_8010D692;
+extern s32 D_8010D694;
+extern s8 D_8010D698;
+void func_800E9894(void);
+s8 func_800E98D4(void);
+void open_status_menu_short(void);
+void func_800E98C4(void);
+
+typedef s32 PopupType;
```

Comment:
We don't need this, `enum PopupTypes` will be `s32` anyway if we decide to use it.